data/campaigns/The_South_Guard/scenarios/07b_Pebbles_in_the_Flood.cfg

```diff
@@ -380,5 +450,13 @@
                 invulnerable=no
             [/status]
         [/modify_unit]
+        [modify_unit]
+            [filter]
+                id="Mal Tera"
+            [/filter]
+            [status]
+                invulnerable=no
+            [/status]
+        [/modify_unit]
```

Comment:
One could use a single modify_unit tag (or macro) to set the status on both leaders, `id` can be a comma separated list.

However, while it may be unlikely, there is no reason why the other Lich should be impossible to kill. Only Mal M'Brin is needed later for the storyline. I'm pretty sure that it was intentional.